art/core/visualizer/visualization_decorators.py

```diff
@@ -34,7 +34,8 @@ def wrapper_visualize_input(*args, **kwargs):
                 function: Decorated function.
             """
             if visualizing_function_in is not None:
-                visualizing_function_in(*args, **kwargs)
+                to_be_passed = args[1:]
```

Comment:
Why do we need this?

Reply:
As now we decorate classes and not instances. `self` is passed as an first arg here. I thinks we shouldn't give access to  the model. That's why I did this.